Braze Demo.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,12 @@
 	objects = {
```

Comment:
nit: You may want to consider removing the space from the name `Braze Demo.xcodeproj/` since it might cause headaches in the future